Instruments/scripts/dataIdCalculator.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# Calculate dataId
+# This file is part of https://github.com/hh-italian-group/h-tautau.
+
+import uproot
+import pandas
+import sys
+
+file = uproot.open('tauTau_tuple.root')
```

Comment:
- pass the file name and the dataId from the command line;
- wrap code that into a function, so it can be reused in the other modules;
- create a column for each data id component;
- it is just dataId without "sync".